.storybook/storybook.css

```diff
@@ -0,0 +1,9 @@
+.sb-show-main.sb-main-padded {
+  /* remove default padding from component area */
+  padding: 0;
+}
+
+#storybook-root {
+  /* set background color according to theme */
+  @apply h-screen bg-gray-50 dark:bg-gray-900;
+}
```

Comment:
Didn't know about this, with this you can remove https://github.com/nethesis/vue-components/pull/12/files#diff-d42715fd3297e575fb61faba39bba1b83739d3fd533a719ccfb0d81f64862b15R25-R28

Reply:
I tried, they are both needed to apply the correct style in both story and docs page